### PR TITLE
beta7.2 fix-2

### PR DESCRIPTION
**Pull request by CS6**
Waiting to be Accepted 改回 In Progress
Replace  Button ViewAll => more information
Replace Title previewPostCapture => Preview
fix layout bug （fix timestamp 與後面時間切割成兩行）
fix title padding （fix 留白不夠）

參考
https://docs.google.com/presentation/d/1lnXs-dMzKT7IiAce4GdE9KLxohg2SFAWdlb-x0yaxtE/edit#slide=id.ga803bd26e4_0_8

### CODE REVIEW

File: src/assets/i18n/en-us.json [hunk 1]
```diff
@@ -52,14 +52,15 @@
   "logout": "Log Out",
   "tags": "Tags",
   "all": "All",
-  "viewAll": "View All",
+  "viewAll": "open more...",
```
Comment (seanwu1105) [inline src/assets/i18n/en-us.json:55]: Move the property to `"message.viewAll"` for consistency. The properties in root is for title which is capitalized to its key.

File: src/assets/i18n/en-us.json [hunk 2]
```diff
@@ -52,14 +52,15 @@
...
+  "moreInformation": "More Information",
   "mediaId": "Media ID",
   "media": "Media",
   "selectContact": "Select Contact",
   "inviteFriend": "Invite Friend",
   "invitationEmail": "Invitation Email",
   "sendPostCapture": "Send PostCapture",
-  "previewPostCapture": "Preview PostCapture",
+  "previewPostCapture": "Preview",
```
Comment (seanwu1105) [inline src/assets/i18n/en-us.json:63]: Remove the property `"preview"`. The properties in root is for title which is capitalized to its key.

File: src/assets/i18n/en-us.json [hunk 3]
```diff
@@ -76,7 +77,7 @@
   "delivered": "Delivered",
   "returned": "Returned",
   "inProgress": "In Progress",
-  "waitingToBeAccepted": "Waiting to be Accepted",
+  "waitingToBeAccepted": "In Progress",
```
Comment (seanwu1105) [inline src/assets/i18n/en-us.json:80]: Remove the property `"waitingToBeAccepted"`. The properties in root is for title which is capitalized to its key.

File: src/assets/i18n/zh-tw.json [hunk 4]
```diff
@@ -53,13 +53,14 @@
...
   "viewAll": "檢視全部",
+  "moreInformation": "更多訊息",
   "mediaId": "媒體 ID",
   "media": "媒體",
   "selectContact": "選擇聯絡人",
   "inviteFriend": "邀請好友",
   "invitationEmail": "邀請電子信箱",
   "sendPostCapture": "發送 PostCapture",
-  "previewPostCapture": "預覽 PostCapture",
+  "previewPostCapture": "預覽",
```
Comment (seanwu1105) [inline src/assets/i18n/zh-tw.json:63]: Remove the duplicated property.

File: src/assets/i18n/zh-tw.json [hunk 5]
```diff
@@ -75,7 +76,7 @@
   "accepted": "已接收",
   "delivered": "已送出",
   "inProgress": "等待中",
-  "waitingToBeAccepted": "等待對方接收中",
+  "waitingToBeAccepted": "等待中",
```
Comment (seanwu1105) [inline src/assets/i18n/zh-tw.json:79]: Change the property according to the suggestion of its English translation.

File: src/assets/i18n/zh-tw.json [hunk 6]
```diff
@@ -53,13 +53,14 @@
   "tags": "標籤",
   "all": "所有",
   "viewAll": "檢視全部",
```
Comment (seanwu1105) [inline src/assets/i18n/zh-tw.json:55]: Change the property according to the suggestion of its English translation.

**Review by seanwu1105 — CHANGES_REQUESTED** [review]

**Comment by seanwu1105** [comment]
@CS6 Please change the target branch to master as this is a patch to the latest release.

**Comment by seanwu1105** [comment]
@CS6 Please also rebase this fix to master branch and drop the commits between the latest develop HEAD and master HEAD as we don't want to release the fix with _Update dependencies. `63e45d6f`_, _Handle geolocation plugin errors and unknown location display (#308) `fe7cbdf6`_ and _Show contact list on contact selection dialog. #307 `fa8ef50`_.

**Review by seanwu1105 — CHANGES_REQUESTED** [review]
@CS6 Please also rebase this fix to master branch and drop the commits between the latest develop HEAD and master HEAD as we don't want to release the fix with _Update dependencies. `63e45d6f`_, _Handle geolocation plugin errors and unknown location display (#308) `fe7cbdf6`_ and _Show contact list on contact selection dialog. #307 `fa8ef50`_.

**Comment by seanwu1105** [comment]
@CS6 Did you drop the three commits I mentioned after rebasing on master? I still see the commits :thinking:

**Comment by CS6** [comment]
@seanwu1105  我這邊看是 rebase 到 master [0.10.0]( c5971d,983a1a)，應該有符合需求？

**Comment by seanwu1105** [comment]
@CS6 rebase 以外，在 master 到 develop 之間有三個 commits 需要被 dropped 避免這個 patch 有新的 features 在裡面。

一開始的 patch 應該是要從 master checkout 就不用像這次還要 rebase 和 drop commits。

**Comment by seanwu1105** [comment]
@CS6 After the discussion on Slack, you do not need to drop the commits. All you need to do is change back the target branch to develop, and then I will merge this fix to the next release (beta 7.3). Thanks.

> Sean Wu  9:56 AM
> @Scott Yan 目前還在等 pr 收起來 https://github.com/numbersprotocol/capture-lite/pull/320
> 不過因為今天要 release 7.3，不知道能不能夠一併釋出就好？（cc @Tammy）
> 
> Tammy  9:57 AM
> @Sean Wu 我原則上希望 UI release 跟 功能 release 分開，因為兩者 QA 要測試的重點不同。不過如果合併比較節省 efforts，可以合併

**Review by seanwu1105 — APPROVED** [review]

**Pull request merged**